packages/pg-protocol/src/b.ts

```diff
@@ -2,21 +2,25 @@
 
 import { Writer } from './buffer-writer'
 import { serialize } from './index'
+import { BufferReader } from './buffer-reader'
```

Comment:
Does this make serialize an unused import?

Reply:
ah yeah this file is trash I'm just using during development, good thinking though.  After I merge your pool PR I'm going to turn on significant linting & prettier on the whole codebase.  

1) some of the code, particularly tests, in the test folder, is older than the hills.  It's mostly valuable but hasn't been linted or prettiered as both eslint and prettier didn't exist when it was written, so been putting it off
2) this repo recently became a mono-repo from several other projects, each w/ slightly different lint rules
3) been busy w/ covid anxiety!

But anyways, yeah huge PR incomming tomorrow (I think tomorrow) to lint everything. until then...chaos reigns (so long as tests pass)